R/mod_gene_var.R

```diff
@@ -25,12 +25,17 @@ mod_gene_variant_ui <- function(id){
       selectizeInput(ns("Genes"), label = "Genes", choices = unique(kairos::jhu_tumor_file@data$Hugo_Symbol),
                      selected = "NF1", multiple = F),
       
+      box(title = "Samples in the selected cohort", 
+          status = "primary", solidHeader = TRUE,
+          width = 1000,
+          collapsible = FALSE,
+          textOutput(ns('sample_check'))),
+      
       box(title = "Positional information of variants in tumor samples", 
           status = "primary", solidHeader = TRUE,
           width = 1000,
           collapsible = FALSE,
-          plotOutput(ns('lollipop_plot'))
-      )
+          plotOutput(ns('lollipop_plot')))
```

Comment:
I get this error: "argument "p" is missing, with no default" with the full cohort selected.

Reply:
hmmm... I am not getting that error message on my end. 

In previous iterations, I got this error when the cohort excluded all samples with genomic var data and the plotting dataset consisted of 0 samples. But in this case the new user messages should take care of this scenario.

Reply:
I found the root of this bug: https://github.com/PoisonAlien/maftools/issues/387

Reply:
awesome! 

also, I was able to reproduce the bug on my end when I pulled the develop branch today. On the gene_var_1 branch its works well but not on the develop :) possibly due to the issue you mentioned above :)

Reply:
Re-opening this conversation since this bug is now bugging me! :(

Reply:
To clarify my not-so-clear remark earlier: Acc to issue 387, the issue of maftools layout being overwritten by plotly was fixed in the latest version. But even with the latest version pulled from github, I am encountering this bug when I run the app locally. I reported this observation to PoisonAlien.